### ISSUE TITLE
[Bug Report][2.5.14] Vuetify tooltip is not read by screen reader(accessibility)

### ISSUE THREAD

**Issue by bobby17jones** [vuetifyjs/vuetify#15402]
### Environment
**Vuetify Version:** 2.5.14
**Vue Version:** 2.6.14
**Browsers:** Mozilla Firefox, Google Chrome
**OS:** Windows 10

### Steps to reproduce
Install any screen reader (Jaws/Screen Reader Google Chrome)  

Click/Tab on the button (with the tooltip)

The screen reader will read only the button but not the tooltip.

### Expected Behavior
The screen reader reads out the tooltip.

### Actual Behavior
The screen reader doesn't read out the content of the tooltip

### Reproduction Link
<a href="https://codepen.io/bobby17jones/pen/KKoVYgg?editors=101" target="_blank">https://codepen.io/bobby17jones/pen/KKoVYgg?editors=101</a>

### Other comments
The correct aria attributes have to be set. The tooltip needs a aria-role="tooltip" and the activator needs a aria-describedby.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role

**Comment by KaelWD** [comment]
This is already fixed in v3.
